docs/decisions/0004-avro-attrs-bridge-implementation-decisions.rst

```diff
@@ -0,0 +1,65 @@
+========================================
+AvroAttrsBridge implementation decisions
+========================================
+.. contents::
+
+Context
+-------
+
+- `0003-events-payload.rst <https://github.com/eduNEXT/openedx-events/blob/main/docs/decisions/0003-events-payload.rst#decisions>`_: Openedx has standardized around using attrs classes to define event data.
+
+- `OEP-52: Event Bus (Draft) <https://github.com/openedx/open-edx-proposals/pull/233>`_:  Openedx is testing out event bus technology for communication between services and within services. We are currently trialing Kafka.
+
+- The industry standard for messaging format used with Kafka is Avro (there are other options, but Avro has been the standard for a while). In decision `open-edx-proposals/oep-0052/001-schema-representation.rst <https://github.com/openedx/open-edx-proposals/blob/7bf9acedae5f4290ac2d0e4374c3078278842801/oeps/architectural-decisions/oep-0052/decisions/001-schema-representation.rst>`_: We decided to create a bridge class to convert from Attrs data class to Avro specification.
+
+The purpose of this ADR is to document decisions made while developing `AvroAttrsBridge <https://github.com/eduNEXT/openedx-events/blob/main/openedx_events/avro_attrs_bridge.py>`_ class.
+
+
+Decision
+--------
+
+Each AvroAttrsBridge class will support:
+
+1. Avro Schema creation is validated at instatiation.
```

Comment:
Nit: the listen items don't actually complete the sentence at the head of the list, which makes it a bit hard to read. I think we can just scrap the heading and label this "Decisions"
Also the verb tenses of the list items should be consistent